rules-tests/TypeDeclaration/Rector/Property/PropertyTypeDeclarationRector/Fixture/private_with_inherit_doc.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\PropertyTypeDeclarationRector\Fixture;
+
+final class A {
+    /**
+     * @inheritDoc
+     */
+    private $property = 'test';
+}
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\PropertyTypeDeclarationRector\Fixture;
+
+final class A {
+    /**
+     * @var string
+     */
+    private $property = 'test';
```

Comment:
this will make second rector run apply the change to typed property, probably just better directly apply the change to typed.